fea-rs/src/compile/lookups/contextual.rs

```diff
@@ -213,7 +213,7 @@ impl ContextualLookupBuilder<SubstitutionLookup> {
                 SubstitutionLookup::Multiple(subtables) => subtables
                     .subtables
                     .iter()
-                    .all(|subt| !subt.contains_target(target)),
+                    .all(|subt| subt.can_add(target, &replacements)),
```

Comment:
why should this reuse of chained inline/anonymous lookups be limited to multiple substitutions? in theory the same treatment could be applied to other GSUB types like single or ligature substituions?

Reply:
we already have this implemented for single sub, matching fonttools. We do not have it implemented for ligature substitutions, although we probably should, but I'd want to know what fonttools is doing and do that in a separate PR with good testing etc.